### ISSUE TITLE
Revise how api docs are generated

### ISSUE THREAD

**Issue by sethvincent** [digidem/mapeo-schema#28]
Right now in #22 the docs being generated are ignored in git in part because the typedoc setup isn't complete. There's also some complexity with having two `tsc` scripts in the build step (one for index.js and one for everything else) that makes it tricky to configure typedoc.

There might be a clean solution by creating a tsconfig.docs.json config file that's just used for typedoc.

Either that or finding a way to have just one `tsc` script might work.

**Comment by tomasciccola** [comment]
Yeah, I haven't figured how to have only one tsc stage. I'll probably look up having a separate tsconfig for typedoc
